Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -845,10 +845,12 @@ getPaymentHistoryEntityDetails merchantShortId driverId invoiceId = do
   Driver.getHistoryEntryDetailsEntityV2 (personId, merchant.id) invoiceId
 
 ---------------------------------------------------------------------
-clearOnRideStuckDrivers :: ShortId DM.Merchant -> Flow Common.ClearOnRideStuckDriversRes
-clearOnRideStuckDrivers merchantShortId = do
+clearOnRideStuckDrivers :: ShortId DM.Merchant -> Maybe Int -> Flow Common.ClearOnRideStuckDriversRes
+clearOnRideStuckDrivers merchantShortId dbSyncTime = do
   merchant <- findMerchantByShortId merchantShortId
-  driverInfos <- B.runInReplica QPerson.getOnRideStuckDriverIds
+  now <- getCurrentTime
+  let dbSyncInterVal = addUTCTime (fromIntegral (- fromMaybe 1 dbSyncTime) * 3600) now
```

Comment:
Lets take this input in minutes intead of hours.

Reply:
ok